Makefile

```diff
@@ -16,14 +16,20 @@ CFLAGS ?= -g -O0
 
 CFLAGS	+= -Wall -Wextra
 
+ifeq ($(UNAME_S),FreeBSD)
+	CFLAGS	+= -lusb
```

Comment:
This doesn't look right. -lxxx should be added only to `LDFLAGS`, when stuff like optimization parameters or include directories should be added to `CFLAGS`.